src/uwtools/api/driver.py

```diff
@@ -24,5 +121,34 @@ def _add_classes():
         __all__.append(classname)
 
 
-__all__: list[str] = []
+def _get_driver_class(
+    classname: str, module: str, module_dir: Optional[Union[Path, str]] = None
+) -> Optional[Type]:
```

Comment:
Here we find the module (either via the path provided or in the CWD), import it, and return the driver class.